templates/createdrive.html

```diff
@@ -71,7 +71,7 @@ <h2>Caption</h2>
     <input class="form-control" id="captionDrive" name="captionDrive" type="text" placeholder="Caption your drive" aria-label="Drive Caption" size="150px">
 <div class="mb-3">
     <h2>Add a photo from your drive</h2>
-    <input class="form-control" type="file" id="drivePhoto" name="drivePhoto">
+    <input class="form-control" id="drivePhoto" name="drivePhoto" type="text" placeholder="Please link to a photo of your drive">
```

Comment:
Did you add this to the editDrive.html as well?

Reply:
I didn't, good catch. Appreciate it.